website/source/docs/commands/alloc/exec.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+layout: "docs"
+page_title: "Commands: alloc exec"
+sidebar_current: "docs-commands-alloc-exec"
+description: >
+Stream the exec of a task.
+---
+
+# Command: alloc exec
+**Alias: `nomad exec`**
+
+The `alloc exec` command runs a command in a running allocation.
+
+## Usage
+
+```
+nomad alloc exec [options] <allocation> <command> [<args>...]
+```
+
+
+This command runs the passed command inside the environment of the given allocation and task.
```

Comment:
I think we need to add a bit of text explaining the use cases/motivation here. perhaps something like:
```
The nomad exec command can be use to run commands inside a running task/allocation. Use cases are for inspecting container state, debugging a failed application without needing ssh access into the node that's running the allocation. 
```